### DISCUSSION TITLE
Two Thirds Centred Option

### DISCUSSION THREAD

**Discussion by pharwood** [rxhanson/Rectangle#502] [Ideas]
Would it be possible to add an option to have a window resize to two thirds size, but centred?

I often resize browser windows to First Two Thirds and then drag to the centre of my screen. Nice size for a browser window.

**Comment by rxhanson** [comment]
In the latest version, thirds cycling is the default behavior for "Center half", so you'd get center 2/3 by executing center half three times.

Alternatively, you could do one of the following:
1. [Convert "Almost maximize" to be center 2/3](https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#almost-maximize)
2. [Add an extra centering command with custom size](https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#add-an-extra-centering-command-with-custom-size)

The shortcuts preference window is maxed out, as Rectangle is meant to be a simple app. 

The other alternative is to use [Rectangle Pro](https://rectangleapp.com/pro) (paid), which has Center 2/3 included.
